common/src/main/java/com/wynntils/utils/render/MapRenderer.java

```diff
@@ -4,6 +4,9 @@
  */
 package com.wynntils.utils.render;
 
+import static com.wynntils.utils.MathUtils.signedArea;
+import static com.wynntils.utils.VectorUtils.lineIntersection;
```

Comment:
I just noticed. I think we agreed on not using static imports (like we don't use import *), but this was not blocked by spotless. Can you refactor?

Reply:
intellij import it by default, but sure